### PR TITLE
#4659 - Use the methods from UiUtil.java

### LINKED ISSUE #4659: Reduce number of warnings related to UserProvider.getCurrent
### Feature Description
UserProvider.getCurrent can return null. Code analysis complains about a possible null pointer exception.
Actually, a NPE is not possible because the UI is SormasUI which always has a non-null user provider

### Proposed Change
Use SormasUI methods to get the user provider

### PR DESCRIPTION
**Pull request by carina29**
Fixes #4659

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

File: sormas-ui/src/main/java/de/symeda/sormas/ui/UiUtil.java [hunk 1]
```diff
@@ -3,16 +3,94 @@
...
+
+	public static JurisdictionLevel getJurisdictionLevel() {
+		return Objects.nonNull(getCurrentUserProvider()) ? getCurrentUserProvider().getJurisdictionLevel() : null;
+	}
+
+	public static String getUserName() {
+		return Objects.nonNull(getCurrentUserProvider()) ? getCurrentUserProvider().getUserName() : null;
+	}
+
+	public static String getUuid() {
```
Comment (leventegal-she) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/UiUtil.java:38]: please rename to `getUserUuid` to be clear what uuid it returns

**Review by leventegal-she — CHANGES_REQUESTED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=13026

**Review by leventegal-she — APPROVED** [review]

**Pull request merged**